modules/vulnerability-management20-download-reports.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * operating/manage-vulnerabilities.adoc
+:_content-type: PROCEDURE
+[id="vulnerability-management20-download-reports_{context}"]
+= Downloading vulnerability reports
+
+[role="_abstract"]
+You can generate an on-demand vulnerability report and then download it.
++
+[NOTE]
+====
+You can only download reports that you have generated; you cannot download reports generated by other users.
+====
+
+.Procedure
+. In the {product-title-short} web portal, navigate to *Vulnerability Management 2.0* -> *Vulnerability Reporting* and locate the report you want to download in the list of reports. Generate the vulnerability report by using one of the following methods:
+* To generate the report from the list:
+.. Click {kebab}.
+.. Select *Generate download*. The *My active job status* column displays the status of your report creation. After the *Processing* status goes away, you can download the report.
+* To generate the report from the report window:
+.. Click the report name to open the report window.
+.. Click *Actions* and select *Generate download*.
+. To download the report, if you are viewing the list of reports, click the report name to open it.
```

Comment:
...if you are viewing a list of report configs, click on config name to open it.